Havit.Blazor.Components.Web.Bootstrap/Forms/Internal/HxMultiSelectInternal.razor.cs

```diff
@@ -14,6 +16,8 @@ public partial class HxMultiSelectInternal<TValue, TItem>
 
 	[Parameter] public List<int> SelectedIndexes { get; set; }
```

Comment:
Please remove the parameter, this is dead code (it was no longer being used before the PR).

Reply:
I have removed this property and the Values property which was also not in use.